src/FakeClient/GraphicsCaptureEventProcessor.h

```diff
@@ -233,7 +233,7 @@ class GraphicsCaptureEventProcessor : public orbit_capture_client::CaptureEventP
   static void UpdateFrameDurationDistribution(uint64_t frame_time_ns, Distribution& distribution) {
     if (frame_time_ns > 0) {
       double frame_time_ms = frame_time_ns / 1.0e6;
-      int distribution_index = std::min(static_cast<int>(frame_time_ms), kMaxTimeMs);
+      int distribution_index = std::max(0, std::min(static_cast<int>(frame_time_ms), kMaxTimeMs));
```

Comment:
This should fix the crash, but I think the exceptional case of `frame_time_ms` being so large that it overflows the int should actually be excluded from the stats. The current code is incorrectly inflating the `0` bucket of the histogram when this exceptional case occurs. I also don't think we need to do the computation in double, as casting to int rounds down anyway (for positive numbers), so I believe that the following would give you the desired result:
```
uint64_t frame_time_ms_floor = frame_time_ns / 1000000;
uint64_t index = std::clamp(frame_time_ms_floor, 0, kMaxTimeMs);
```

Reply:
Or, if you want to exclude the time (probably better), return if `frame_time_ms_floor > kMaxTimeMs`. Maybe log a warning in that case that mentions the number of times excluded because of this reason.

Reply:
So I changed to skip the ones frame_time_ms > kMaxTimeMs and also print their command buffer timestamps. 
I wasn't sure if skipping it's better or not but since it's reported in the log it's fine

Reply:
I think skipping is better, otherwise we are mixing values that are too large with those that actually belong into the last bucket in the histogram. But I suppose it depends on whether this is an exceptional case that, when it happens, means that the run is essentially broken and the data is useless, or whether it is a case that happens on many runs, but only rarely (such on boundaries of the timing data). In either case, I think it's cleaner to separate the actual useful data from what we consider garbage.